vtds_application_openchami/private/templates/nginx-default-site-config

```diff
@@ -0,0 +1,24 @@
+# Default server configuration
+#
+server {
+        # SSL configuration
+        #
+        listen 443 ssl;
+        listen [::]:443 ssl;
+        server_name _;
+
+        ssl_certificate /etc/sushy-emulator/cert.pem;
+        ssl_certificate_key /etc/sushy-emulator/key.pem;
```

Comment:
Just curious how are these getting populated? Should there be a self-signed ssl cert generator call somewhere?

Reply:
In the deploy script for the blade (blade_deploy.sh) I generate the self-signed cert and its key and stuff it away in that directory along with the sushy-emulator configuration.